tango_simlib/examples/override_class.py

```diff
@@ -490,3 +495,240 @@ def test_action_stoprainstorm(self, model, tango_dev=None, data_input=None):
         else:
             quant_rainfall.max_bound = float(quant_rainfall.meta['max_bound'])
             quant_rainfall.mean = 1.00
+
+
+class OverrideDish(object):
```

Comment:
So these are just stubs that you will expand later?

Also, I hope the nice docstrings here are also in the .xmi file for the commands so that they can be read in a TANGO UI?